### PR TITLE
incorp greenelab eslint config

### PR DESCRIPTION
**Pull request by vincerubinetti**
- install eslint-config-greenelab and prettier-config-greenelab
- relint entire codebase

### CODE REVIEW

**Comment by vincerubinetti** [comment]
Create react app not respecting imported eslint rules for build per [this issue](https://github.com/facebook/create-react-app/issues/8683). Going to try reverting CRA version to fix the build error.

**Comment by vincerubinetti** [comment]
Nevermind, that upstream problem was not the issue (at least this time). I forget to put `EXTEND_ESLINT=true` in `.env`.

Good to look out for with CRA `3.4.1` though, since it's a caching issue and could appear without warning.

File: src/util/object.js [hunk 1]
```diff
@@ -64,6 +64,7 @@ export const filterKeys = (object, filter = [], whiteList = false) => {
       delete object[key];
   }
 
+
```
Comment (dongbohu) [inline src/util/object.js]: Is this blank line added on purpose?

**Review by dongbohu — APPROVED** [review]
One comment only. Feel free to close it.

**Pull request merged**